elasticdl/python/elasticdl/layers/embedding.py

```diff
@@ -140,7 +141,7 @@ def _check_id_valid(self, ids):
                 % (first_may_exceed_id, self.input_dim)
             )
         if any(ids < 0):
-            raise ValueError(
+            logger.warning(
```

Comment:
It's better that we can make it configurable.

Reply:
But, how and where to config it? `tf.nn.safe_embedding_lookup_sparse` auto prunes invalid IDs ( < 0) and `tf.nn.embedding_lookup_sparse` will raise an error with  invalid IDs ( < 0).